src/store/typings.ts

```diff
@@ -109,3 +109,11 @@ export enum ERaceEnum {
   UNDEAD = 8,
   TOTAL = 16,
 }
+
+export enum ERaceEnumPretty {
+  Random = 0,
```

Comment:
Remove this and use $t. Races are already in the locale:
https://github.com/modmoto/w3champions-ui/blob/master/src/locales/en.ts#L10